src/TaskoPhobia.Infrastructure/Auth/Authenticator.cs

```diff
@@ -11,12 +11,12 @@ namespace TaskoPhobia.Infrastructure.Auth;
 
 public class Authenticator : IAuthenticator
 {
-    private readonly IOptions<AuthOptions> _options;
-    private readonly TimeSpan _expiry;
     private readonly string _audience;
+    private readonly TimeSpan _expiry;
     private readonly string _issuer;
+    private readonly JwtSecurityTokenHandler _jwtSecurityTokenHandler = new();
+    private readonly IOptions<AuthOptions> _options;
     private readonly SigningCredentials _signingCredentials;
-    private readonly JwtSecurityTokenHandler _jwtSecurityTokenHandler = new ();
 
     public Authenticator(IOptions<AuthOptions> options)
```

Comment:
polecam korzystać z IOptionsMonitor albo ioptionssnapshot (jeden to scoped a drugi to transient). Dlaczego korzystać z nich? Bo samo IOptions zaciąga zmiany raz na start apki i tyle, a te 2 wcześniejsze zwrócą nowe dane jeśli sie zaaktulizowały